lib/core/util.js

```diff
@@ -417,10 +409,7 @@ function throwIfAborted (signal) {
     signal.throwIfAborted()
```

Comment:
throwIfAborted was added in node v17.3.0 according to MDN. That branch can't be reached anymore, so the entire thing can be replaced with just signal.throwIfAborted